### DISCUSSION TITLE
The new image manipulation area in the latest version has a conflict issue with right-click image movement and mouse gestures.

### DISCUSSION THREAD

**Discussion by strongshuai** [lllyasviel/stable-diffusion-webui-forge#1898] [General]
![屏幕截图 2024-09-23 164441](https://github.com/user-attachments/assets/0f9e924f-856a-49b6-8e82-c01e6c524934)
The new image manipulation area in the latest version has a conflict issue with right-click image movement and mouse gestures. Nowadays, most browsers also use right-click for mouse gestures. However, the updated version of Forge has changed to using right-click to drag images, causing a significant conflict with mouse gestures. Is there a more reasonable solution to this problem?

**Comment by Kmolpat** [comment]
And I'm unable to draw with a pen tablet.
